### DISCUSSION TITLE
可否添加配置项让横幅占整个屏幕，然后点击下拉按钮就能跳到下面的文章页面

### DISCUSSION THREAD

**Discussion by IshuaWang** [hcjike/halo-theme-dream2.0-plus#36] [General]
如果可以兼容这样就比较完美了，有的人喜欢这种风格，就像下面这种
![2d2384f0ff66fd159b3dcb3051f69f6a](https://github.com/user-attachments/assets/0441880f-40a9-49e6-8cd4-c90d02a7b272)

**Comment by mjsoftking** [comment]
这不是这套主题的风格，如有需要可以fork自行修改
